understanding/22/accessible-authentication.html

```diff
@@ -126,7 +126,7 @@ <h3>Sufficient Techniques for Accessible Authentication</h3>
                 <a href="../../techniques/general/G218.html" class="general">Email link authentication</a>
             </li>
             <li>
-                Providing a properly marked up email and password inputs (Potential future technique)
+                <a href="../../techniques/html/H100.html" class="html">Providing a properly marked up email and password inputs</a>
```

Comment:
remove the "a": "Providing *a* properly marked up email and password inputs" > "Providing properly marked up email and password inputs"